src/Core/ServiceWrapper_dotNET4/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="ICSharpCode.SharpZipLib.dll" version="0.85.4.369" targetFramework="net40" />
   <package id="ILMerge" version="2.14.1208" targetFramework="net20" />
   <package id="log4net" version="2.0.8" targetFramework="net20" requireReinstallation="True" />
   <package id="MSBuildTasks" version="1.4.0.88" targetFramework="net20" />
-</packages>
+</packages>
```

Comment:
hmm, would be better to revert

Reply:
Need the library ICSharpCode.SharpZipLib.dll to carry out zipping as core is still using Framework 2.0 and the Zip classes are only available from .Net Framework 4.5 +